app/Child.php

```diff
@@ -137,4 +149,33 @@ public function family()
     {
         return $this->belongsTo('App\Family');
     }
+
+    /**
+     * Get info about whether child can defer
+     *
+     * @return boolean
+     */
+    public function getCanDeferAttribute()
+    {
+      $notices = $this->family->registrations[0]->getEvaluator()->evaluations["App\Child"]["notices"];
+      if (!array_key_exists('ScottishChildCanDefer', $notices)) {
+        return false;
+      }
+      $rule = [
+        new Evaluation([
+              "name" => "ScottishChildCanDefer",
+              "value" => 0,
+              "purpose" => "notices",
+              "entity" => "App\Child",
+        ])
+      ];
+      $rulesMods = collect($rule);
+      $evaluator = EvaluatorFactory::make($rulesMods);
+      $evaluation = $evaluator->evaluate($this);
+      $notices = $evaluation["notices"];
+      if (count($notices)) {
```

Comment:
Probably should check of `count($notices) > 0`
or ternary to 

`return (count($notices) > 0) ? true : false;`

but could also be cast to a bool

`return (bool) count($notices);`

Reply:
Also, check that the notice is explicity the ScottishChildCanDefer one
Otherwise, we assume that the default notices the EvaluatorFactory holds will never be altered to inlude _other_ notices about a child

Reply:
Done.